lib/msf/ui/console/command_dispatcher/modules.rb

```diff
@@ -426,7 +438,19 @@ def cmd_search(*args)
               else
                 search_params = Msf::Modules::Metadata::Search.parse_search_string(match)
                 @module_search_results = Msf::Modules::Metadata::Cache.instance.find(search_params)
-              end
+
+                if sort and sort_options.include?(sort)
+                  if sort == 'date'
+                    sort='disclosure_date'
+                  end
+                  eval("@module_search_results.sort_by! {|meta| meta.#{sort}}")
+                end
+
+                  if desc
```

Comment:
Indention here should be aligned with indention of the earlier `if...end` block. Recommend running `rubocop -a` to solve these indention issues automatically.